base/server/python/pki/server/cli/audit.py

```diff
@@ -379,6 +380,83 @@ def execute(self, argv):
             print('  Filter: %s' % event.get('filter'))
 
 
+class AuditEventShowCLI(pki.cli.CLI):
+
```

Comment:
Probably a Doc text with a little explanation of this class would address my following comment

Reply:
I think the wiki page above would provide the explanation. It will be difficult to explain the whole concept in a Doc text.